src/components/layout/nav.js

```diff
@@ -116,7 +116,16 @@ export const Wrapper = styled(Container)`
     @media ${device.laptop} {
         font-size: var(--text-size-xxs);
     }
+    @media ${device.mobileM} {
+        ${({ offset_px_mobile }) =>
+            offset_px_mobile ? `width: calc(100% - ${offset_px_mobile}px)` : ``};
```

Comment:
```suggestion
        ${({ offset_px_mobile }) => offset_px_mobile && `width: calc(100% - ${offset_px_mobile}px)`};
```

Reply:
IMO, I think we should write it in this manner as a form of better practice because if no prop was passed, we wouldn't want to return `width: ''` as an invalid CSS before falling back to whatever value it was before.

Reply:
it won't return `width: ''` as the whole expression is wrapped after the condition. But we can do like this to maintain consistency.